README.md

```diff
@@ -122,3 +122,5 @@ This project is authored by the Home Office.
 ### License 
 
 This project is licensed under the MIT license. For details please see [License](LICENSE) 
+
+This project Contains public sector information licensed under the Open Government Licence v3.0. (http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)
```

Comment:
```suggestion
This project contains public sector information licensed under the Open Government Licence v3.0. (http://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/)
```

Add this to the LICENSE file too. Just a "Contains public sector information licensed under the Open Government Licence v3.0".